3-JSS/src/styles/styles.css

```diff
@@ -0,0 +1,79 @@
+
+:root {
+    --main-gradient-color: linear-gradient(to bottom,white, blue);
```

Comment:
loved your gradient :). however I noticed a white margin in the body (hint). plz fix